client/windows/GUIClasses.cpp

```diff
@@ -787,9 +787,18 @@ void CTavernWindow::HeroPortrait::hover(bool on)
 		GH.statusbar->clear();
 }
 
+static const char *NEW_EXCHANGE_BG = "TRADE2X";

```

Comment:
"NewSmth" is not a good name, these features are new up until this PR is merged.